miso-web/src/main/webapp/scripts/list_assay.js

```diff
@@ -6,37 +6,54 @@ ListTarget.assay = (function ($) {
     getUserManualUrl: function () {
       return Urls.external.userManual("requisitions", "assays");
     },
+    createUrl: function (config, projectId) {
+      throw new Error("Must be provided statically");
+    },
+    getQueryUrl: null,
     showNewOptionSop: true,
     createBulkActions: function (config, projectId) {
-      return !config.isAdmin
-        ? []
-        : [
-            {
-              name: "Copy",
-              action: function (items) {
-                if (items.length > 1) {
-                  Utils.showOkDialog("Error", ["Select an individual assay to copy"]);
-                  return;
-                }
-                window.location = Urls.ui.assays.create + "?" + $.param({ baseId: items[0].id });
-              },
+      if (config.admin) {
```

Comment:
* Why did this change from `isAdmin` to `admin`?
* The admin options should not be included on the project page